packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js

```diff
@@ -20,9 +20,9 @@ const setCookie = (named, value, req, res) => {
 			secure: true,
 			httpOnly: false
 		})
-		return cookies.set(named, value)
+		return cookies.set(named, value, { sameSite: 'None' })
```

Comment:
Does the case of `'None'` matter? It's uppercase here but lowercase in `auth.js`

Reply:
Nope it doesn't, but I'll make it upper-case for consistency.